### DISCUSSION TITLE
Which cache should be shared among loadModule calls?

### DISCUSSION THREAD

**Discussion by joepaolicelli** [FranckFreiburger/vue3-sfc-loader#135] [Q&A]
[In the API docs](https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/api/README.md#loadmodule):
> If for some reason you need to use it in your components, be sure to share at least the options.compiledCache object between all calls

[In the FAQ](https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/faq.md#can-i-call-loadmodule-several-times-):
> the application must, at least, share the same options.moduleCache through all its loadModule() calls

Which one do I actually need to share? Or is it both? Or maybe moduleCache always needs to be shared, but compiledCache only needs to be shared when called from child components? Has anyone had any experience with this? Thanks!

**Comment by FranckFreiburger** [comment]
Ambiguity has been cleared up in doc.
`moduleCache` must be shared between all `loadModule()` calls.
